ieee754r/nextupd64.c

```diff
@@ -0,0 +1,29 @@
+/* Returns the next representable neighbor of x in the direction of y
+
+   Copyright (C) 2006 IBM Corporation.
+   Copyright (C) 2007-2015 Free Software Foundation, Inc.
```

Comment:
This should be updated to 2019. I don't think the IBM copyright is relevant to newly added files.

Reply:
https://github.com/libdfp/libdfp/blob/21b9650b3c8015da74ef360cb86111864c08dc91/ieee754r/nextupd64.c#L3-L7